client/app/payment-form.component.ts

```diff
@@ -1,16 +1,17 @@
-import { Component, OnInit } from '@angular/core';
-import { FormControl, FormGroup, Validators } from '@angular/forms';
+import {Component, OnInit} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
 
-import { combineLatest, Observable, of, throwError } from 'rxjs';
-import { map, switchMap, take } from 'rxjs/operators';
-import { Payment } from 'rightfoot-node/1-3/api';
+import {combineLatest, Observable, of, throwError} from 'rxjs';
+import {map, switchMap, take} from 'rxjs/operators';
+import {Payment} from 'rightfoot-node/1-3/api';
 
-import { UnitedStatesTerritories } from './us-states-and-territories';
-import { phoneNumberValidator } from './phone-number.validator';
-import { RightfootApiService } from './rightfoot-api.service';
-import { DemographicsFormValue } from './demographics-form-value.interface';
-import { StorageService } from './storage.service';
-import { PlaidService } from './plaid.service';
+import {UnitedStatesTerritories} from './us-states-and-territories';
+import {phoneNumberValidator} from './phone-number.validator';
```

Comment:
Please, revert the changes that remove spaces within import brackets. Perhaps your IDE is not configured to preserve/add them. I see that this happens through all files.

Reply:
Corrected! Added tslint rule to catch this in the upstream